### DISCUSSION TITLE
Exit codes in WP-CLI ?

### DISCUSSION THREAD

**Discussion by kanlukasz** [wp-cli/wp-cli#5721] [How do I?] [answered]
Hello

Can we generally say that every command in wp-cli returns:
- Success exits as `0`
- Error exits as `1`

??

Are there other exit codes in other cases?

I need to write some scripts so I would like to know how return codes work

**Comment by danielbachhuber** [comment, marked as answer]
Thanks for the question, @kanlukasz!

Yes, success exits as `0` and error exits as `1`.

Here's an issue with some relevant history: https://github.com/wp-cli/wp-cli/issues/4525
